public/theme/extensions-2021/partials/_master.twig

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang={{ htmllang() }}>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+          rel="stylesheet">
+
+    <link rel="stylesheet" type="text/css" href="{{ asset('css/app.css', 'theme') }}">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="shortcut icon" href="{{ asset('img/favicon.ico', 'theme') }}">
+
+    <!-- Matomo -->
```

Comment:
Can you confirm if this is the right code for the extensions page?

Each site/subsite gets its own Matomo tracking code 👍

Reply:
No, I have deleted it